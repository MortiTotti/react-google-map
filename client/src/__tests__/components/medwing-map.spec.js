import React from 'react';
import { MedwingMap } from 'Components';
import Enzyme, { shallow, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe('Medwing Map component', () => {

    const mockCenter = { lng: 51.0, lat: 52.0 }
    const mockMarkers = [];

    it('should render correctly', () => {
        const component = shallow(
            <MedwingMap center={mockCenter} markers={mockMarkers} />
        );

        expect(component).toMatchSnapshot();
    });
});