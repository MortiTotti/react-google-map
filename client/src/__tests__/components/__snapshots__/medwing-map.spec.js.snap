// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Medwing Map component should render correctly 1`] = `ShallowWrapper {}`;
